tensornetwork/network_components.py

```diff
@@ -1476,11 +1476,12 @@ def _contract_trace(edge: Edge, name: Optional[Text] = None) -> BaseNode:
 def contract(edge: Edge,
              name: Optional[Text] = None,
              axis_names: Optional[List[Text]] = None) -> BaseNode:
-  """
-  Contract an edge connecting two nodes.
+  """Contract an edge connecting two nodes.
+
   All edges of `node1` and `node2` are passed on to the new node,
   and `node1` and `node2` get a new set of dangling edges.
-  `edge is disabled before returning.
+  `edge` is disabled before returning.
+
   Args:
     edge: The edge contract next.
```

Comment:
Should this just read "The edge to contract."?

Reply:
Yes. Done